runtime/vm/resolvesupport.cpp

```diff
@@ -1607,21 +1607,21 @@ resolveVirtualMethodRefInto(J9VMThread *vmStruct, J9ConstantPool *ramCP, UDATA c
 		Trc_VM_resolveVirtualMethodRef_lookupMethod(vmStruct, method);
 
 		/* If method is NULL, the exception has already been set. */
-		if (method != NULL) {
+		if (NULL != method) {
 #if defined(J9VM_OPT_VALHALLA_NESTMATES)
 			J9ROMMethod* romMethod = J9_ROM_METHOD_FROM_RAM_METHOD(method);
 			/* Only allow non-interface method to call invokePrivate, private interface method should use "invokeInterface" bytecode
 			 * The else case will throw ICCE for private interface method 
 			 */
 			if (J9_ARE_ALL_BITS_SET(romMethod->modifiers, J9AccPrivate) && J9_ARE_NO_BITS_SET(resolvedClass->romClass->modifiers, J9AccInterface)) {
 				/* Private method found, will not be in vTable, point vTable index to invokePrivate */
-				if (ramCPEntry != NULL) {
-					ramCPEntry->method = method;
+				if (NULL != ramCPEntry) {
 					UDATA methodIndexAndArgCount = J9VTABLE_INVOKE_PRIVATE_OFFSET << 8;
+					ramCPEntry->method = method;
```

Comment:
Why reorder this line?

Reply:
The intention was to declare variables at top of the scope though I understood that is required for `C` code only.
Restored the order.